universum/modules/automation_server/jenkins_server.py

```diff
@@ -18,9 +20,12 @@ class JenkinsServerForTrigger(BaseServerForTrigger):
     @staticmethod
     def define_arguments(argument_parser):
         parser = argument_parser.get_or_create_group("Jenkins variables", "Jenkins-specific parameters")
-        parser.add_argument('--jenkins-trigger-url', '-jtu', dest='trigger_url',
-                            help='Url to trigger, must include exactly one conversion specifier (%%s) to be '
-                                 'replaced by CL number, for example: http://localhost/%%s', metavar="URL")
+        parser.add_argument('--jenkins-trigger-url', '-jtu', dest='trigger_url', metavar="TRIGGER_URL",
+                            help='Url to trigger, ending with "build" or "buildWithParameters" accordingly')
+        parser.add_argument('--jenkins-trigger-parameter', "-jtp", action="append", nargs='+', dest="param_list",
```

Comment:
Why we need to pass parameter explicitly?
User can already include them into trigger url.